src/screens/Profile/Sections/Feed.tsx

```diff
@@ -42,6 +43,10 @@ export const ProfileFeedSection = React.forwardRef<
   const queryClient = useQueryClient()
   const [hasNew, setHasNew] = React.useState(false)
   const [isScrolledDown, setIsScrolledDown] = React.useState(false)
+  const shouldUseAdjustedNumToRender = feed.endsWith('posts_and_author_threads')
```

Comment:
We are only concerned with doing this when it's the user's "Posts" tab. Subsequent tab switches:

1. May be done after the user has scrolled down the page, and so using the header offset wouldn't provide an accurate number
2. Won't be as noticeably expensive, since the rest of the screen's content has already been rendered.